src/main/scala/com/github/unchama/seichiassist/minestack/MineStackObject.scala

```diff
@@ -0,0 +1,48 @@
+package com.github.unchama.seichiassist.minestack
+
+import org.bukkit.Material
+import org.bukkit.inventory.ItemStack
+
+case class MineStackObject(
+  mineStackObjectName: String,
+  uiName: Option[String],
+  itemStack: ItemStack,
+  hasNameLore: Boolean, // 多分記名があるかどうか
+  category: MineStackObjectCategory
+) {
+
+  def material: Material = itemStack.getType
+
+  def durability: Short = itemStack.getDurability
+
+}
```

Comment:
```suggestion
class MineStackObject(
  val mineStackObjectName: String,
  val uiName: Option[String],
  private val _itemStack: ItemStack,
  val hasNameLore: Boolean, // 多分記名があるかどうか
  val category: MineStackObjectCategory
) {

  def itemStack: ItemStack = _itemStack.clone()

  def material: Material = itemStack.getType

  def durability: Short = itemStack.getDurability

}
```

Bukkit の `ItemStack` は可変オブジェクトなので、クローンして返すべき